### ISSUE TITLE
Continue with technically required cookies only button doesn't unset other cookies

### ISSUE THREAD

**Issue by manucoffin** [porscheofficial/cookie-consent-banner#3]
**Node/npm version(s)**: 
Node: 14.18.0
npm: 6.14.15

**Package(s) version(s)**:

```
    @porscheofficial/cookie-consent-banner@2.1.0
    @porscheofficial/cookie-consent-banner-react@2.1.0
```

**Browser(s) version(s)**:
Google Chrome: 97.0.4692.71

**Description of the issue/feature**:
When accepting all the cookies, then showing the banner again, and clicking on "Continue with technically required cookies only", all the cookies previously set are still set.

For example, on my first visit, I select marketing cookies, and save my preferences. Then, I want to update my choices, so I show the banner, I click on "Continue with technically required cookies only", but the marketing cookies are still set.

**Some example to reproduce the issue**:  
https://codesandbox.io/s/issue-cookie-consent-banner-g48uu

**Steps to reproduce the issue**:
- Click "Agree and continue" to select all the cookie types
- Click on "Show Cookies Banner" button to show the banner
- Click "Continue with technically required cookies only"
- The optional cookies are still set

**Comment by ppuritscher** [comment]
Hi @manucoffin! Thanks for reporting.
Should be fixed with `v2.1.1`, can you confirm that?

**Comment by manucoffin** [comment]
It works perfectly, thanks a lot!
